google-cloud-spanner/src/main/java/com/google/cloud/spanner/connection/ReadOnlyTransaction.java

```diff
@@ -136,6 +137,18 @@ public Timestamp getCommitTimestampOrNull() {
     return null;
   }
 
+  @Override
+  public CommitResponse getCommitResponse() {
```

Comment:
Now there are three instances where this method doesn't make sense. 

Maybe you can construct a commit response for these cases where the server doesn't return one, but maybe the API isn't in the right place.

Reply:
See above.